src/Compilers/Server/VBCSCompilerTests/ServerUtil.cs

```diff
@@ -80,15 +84,29 @@ internal static BuildPaths CreateBuildPaths(string workingDir, string tempDir)
                 tempDir: tempDir);
         }
 
+        /// <summary>
+        /// Gets the client directory from a host if it has one, or a default otherwise
+        /// </summary>
+        private static string GetClientDirectory(ICompilerServerHost compilerServerHost)
+        {
+            var compilerHostImpl = compilerServerHost as CompilerServerHost;
+            return compilerHostImpl?.ClientDirectory ?? DefaultClientDirectory;
+        }
+
         internal static ServerData CreateServer(
-            string pipeName = null,
+            // This is the "uniquefied" token ingredient for pipe name.
```

Comment:
If comments on parameters are necessary please use an XML doc comment and use a `<param>` tag.